From: Randy Dunlap <rdunlap@infradead.org>
Date: Sun, 8 Dec 2019 20:25:10 -0800
Subject: Documentation: x86: fix boot.rst warning and format
Patch-mainline: v5.6-rc1
Git-commit: 0854cbdb1829413680cc1bf072dc68254a5ffe7b
References: jsc#SLE-16407

Fix a Sphinx documentation format warning by breaking a long line
into 2 lines.

Also drop the ':' usage after the Protocol version numbers since
other Protocol versions don't use colons.

Documentation/x86/boot.rst:72: WARNING: Malformed table.
Text in column margin in table line 57.

Fixes: 2c33c27fd603 ("x86/boot: Introduce kernel_info")
Fixes: 00cd1c154d56 ("x86/boot: Introduce kernel_info.setup_type_max")
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
Cc: Thomas Gleixner <tglx@linutronix.de>
Cc: Ingo Molnar <mingo@redhat.com>
Cc: Borislav Petkov <bp@alien8.de>
Cc: "H. Peter Anvin" <hpa@zytor.com>
Link: https://lore.kernel.org/r/c6fbf592-0aca-69d9-e903-e869221a041a@infradead.org
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 Documentation/x86/boot.rst |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/Documentation/x86/boot.rst
+++ b/Documentation/x86/boot.rst
@@ -69,11 +69,12 @@ Protocol 2.13	(Kernel 3.14) Support 32-
 		xloadflags to support booting a 64-bit kernel from 32-bit
 		EFI
 
-Protocol 2.14:	BURNT BY INCORRECT COMMIT ae7e1238e68f2a472a125673ab506d49158c1889
+Protocol 2.14	BURNT BY INCORRECT COMMIT
+                ae7e1238e68f2a472a125673ab506d49158c1889
 		(x86/boot: Add ACPI RSDP address to setup_header)
 		DO NOT USE!!! ASSUME SAME AS 2.13.
 
-Protocol 2.15:	(Kernel 5.5) Added the kernel_info.
+Protocol 2.15	(Kernel 5.5) Added the kernel_info and kernel_info.setup_type_max.
 =============	============================================================
 
 .. note::
